api/dp-openapi.yaml

```
@@ -1290,6 +1290,49 @@ paths:
               schema:
                 $ref: '#/components/schemas/round_stats'
 
+  /documents:
+    get:
+      tags:
+      - documents
+      description: Gets an array of document names, if lang_code is given then only those available in that language
+      parameters:
+      - name: lang_code
+        in: query
+        description: language code
+        schema:
+          type: string
```

This name and description isn't going to be useful, we can help them out:
```suggestion
      - name: language_code
        in: query
        description: Two-letter (ISO 639-1) language code. 
        schema:
          type: string
```